assets/thanos-querier/deployment.yaml

```diff
@@ -46,13 +46,19 @@ spec:
         - --query.replica-label=prometheus_replica
         - --query.replica-label=thanos_ruler_replica
         - --store=dnssrv+_grpc._tcp.prometheus-operated.openshift-monitoring.svc.cluster.local
+        - --query.auto-downsampling
```

Comment:
any concerns regarding enabling auto-downsampling?

Reply:
I don't have any, it should reduce the load in some situations.